openshift/ci-operator/knative-test-images/helloworld/Dockerfile

```diff
@@ -1,4 +1,8 @@
 FROM openshift/origin-base
 
 ADD helloworld /usr/bin/helloworld
+
+RUN mkdir /ko-app && \
+    ln -s /usr/bin/helloworld /ko-app/helloworld
+     
```

Comment:
hmm, I see that this is being aligned with other binaries, do we want to change the entrypoint below as well?

Reply:
There was failed service options test from the previous run.The `--cmd` test defines entrypoint as `/ko-app/helloworld`. In fact we can mimic the upstream completely and copy the binary to `ko-app` dir directly and use it in entrypoint without addition steps in between. Wdyt?

```
      120.582s Revision "svc5-xqvxb-1" failed with message: container create failed: time="2020-09-02T13:09:29Z" level=error msg="container_linux.go:348: starting container process caused \"exec: \\\"/ko-app/helloworld\\\": stat /ko-app/helloworld: no such file or directory\""
```

Reply:
E.g.:
```
FROM openshift/origin-base

ADD helloworld /ko-app/helloworld

ENTRYPOINT ["/ko-app/helloworld"]
```

Reply:
+1 for the adding ko-app based path to entrypoint directly

Reply:
Done.